test/debug/variable_inspector_test.rb

```diff
@@ -227,5 +252,15 @@ def initialize(x:, y:)
         @y = y
       end
     end
+
+    class ClassWithCustomDebugRepresentation
+      def initialize(debug_representation)
+        @debug_representation = debug_representation
+      end
+
+      def debug_representation
```

Comment:
Perhaps it should take unused rest args, in case any need to be added in the future?

```suggestion
      def debug_representation(*, **)
```